### ISSUE TITLE
Some titles show up multiple times

### ISSUE THREAD

**Issue by PatrickvL** [guivanrv/cxbx-reloaded-compatibility-list#1]

**Comment by guivanrv** [comment]
Yeah, I can confirm it, for example for Baldur's Gate: Dark Alliance. 
There are two unique issues though. What would you advise on choosing the right row to show? I think about grouping by ID+Name and in case there're duplicates - showing the one with the most recent activity. Any objections?

**Comment by PatrickvL** [comment]
At least, leave out closed issues.
Duplicates are only those (for now) that have the same title, title ID and region flags (anything else is unique)
